model-optimizer/extensions/back/blob_normalizer.py

```diff
@@ -36,8 +36,7 @@ class BlobNormalizer(BackReplacementPattern):
     enabled = True
 
     graph_condition = [
-        lambda graph: graph.graph['cmd_params'].blobs_as_inputs or
-                      graph.graph['cmd_params'].generate_experimental_IR_V10
+        lambda graph: graph.graph['cmd_params'].blobs_as_inputs
```

Comment:
We should delete blobs_as_inputs as well. IRV10 can not contain blobs. The question is -- if we would like to do it here.